api/extension-guides/language-model.md

```diff
@@ -27,6 +27,7 @@ The following sections provide more details on how to implement these steps in y
 
 - [Chat extension sample](https://github.com/microsoft/vscode-extension-samples/tree/main/chat-sample)
 - [LanguageModels API](https://github.com/microsoft/vscode/blob/5d6671dacb9d6a582b9354ea317211a8e2b2f918/src/vscode-dts/vscode.d.ts#L19190)
+- [@vscode/prompt-tsx npm package](https://www.npmjs.com/package/@vscode/prompt-tsx)
```

Comment:
Do we also need to link to the docs at the top?
I love that we mention it in the Build section and related content.